bottlecap/src/lifecycle/invocation/processor.rs

```diff
@@ -902,12 +902,11 @@ mod tests {
 
         let context = p.context_buffer.get(&request_id).unwrap();
 
-        assert_eq!(
-            context
+        assert!(
```

Comment:
clippy got mad about this?

Reply:
maybe `assert_ne` is better than `!false` on `assert`?